### PR TITLE
feat: add property to user record indicating that it was exported

### LINKED ISSUE #8: add property to user record indicating that it was exported

### PR DESCRIPTION
**Pull request by dviramontes**
closes #8

### CODE REVIEW

File: src/copa_services/core.cljs [hunk 1]
```diff
@@ -68,13 +70,14 @@
                            (if (nil? user)
                              ;; firabase has no record
                              (if (= body "YES")
-                               (-> (PUT url {:created (.now js/Date)})
+                               (-> (PUT url {:archived false
+                                             :created  (.now js/Date)})
                                    (.then #(make-sms twiml (:step-1 outgoing-messages))))
-                               (make-sms twiml (:step-0 outgoing-messages)))
+                               (make-sms twiml(:step-0 outgoing-messages)))
```
Comment (eemshi) [inline src/copa_services/core.cljs]: don't we need the space here (2 arguments) ?

File: src/copa_services/core.cljs [hunk 2]
```diff
@@ -50,6 +50,8 @@
   (let [opts {:url url :method "PUT" :json true}]
     (http (clj->js (assoc opts :body body)))))
 
+;; define a constant set of user properties
+(defonce USER-PROPS #{:created :name :lang :archived})
```
Comment (eemshi) [inline src/copa_services/core.cljs:54]: 👍

File: src/copa_services/core.cljs [hunk 3]
```diff
@@ -89,34 +92,49 @@
...
-                                         body)))))))))))
\ No newline at end of file
+                          (if (>= new-member-count 1)
+                            (-> mailgun
+                                (.messages)
+                                (.send (clj->js data)
+                                       (fn [err body]
+                                         (if-not err
+                                           (update-archived-flag json-user-records filter-archived)
+                                           (js/console.log err)))))
```
Comment (eemshi) [inline src/copa_services/core.cljs]: delete console logs

File: src/copa_services/core.cljs (hunk 1 above)
Comment (dviramontes) [reply src/copa_services/core.cljs]: thanks @eemshi, nice catch!

File: src/copa_services/core.cljs [hunk 4]
```diff
@@ -89,34 +92,49 @@
...
                               attch (.-Attachment mailgun)
                               attachment (attch. (clj->js {:data     (js/Buffer. csv)
                                                            :filename "members.csv"}))
-                              memeber-count (count users)
+                              new-member-count (count filter-archived)
                               data {:from       EMAIL_FROM
                                     :to         EMAIL_TO
-                                    :subject    (str "Hello, " memeber-count " new members this week.")
+                                    :subject    (str "Hello, " new-member-count " new members this week.")
                                     :text       "Exported: daily."
```
Comment (eemshi) [inline src/copa_services/core.cljs]: if we start with daily, it should say "new members today"

**Pull request merged**